.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,29 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Check Prettier
+
+# Controls when the action will run. Triggers the workflow on pull request
+on:
+  pull_request:
```

Comment:
It should be on push not on pull_request. We just want it when changes are made to the branch.